### ISSUE TITLE
storageType

### ISSUE THREAD

**Issue by lars-steffen** [blw-ofag-ufag/eCH-0262#12]
Consider the following topics in the context of storage:
- Feldrandkompostierung
- Temporäre Feldlagerung

**Comment by AFoletti** [comment]
Potentially a minor change

**Comment by hurni** [comment]
requirement unclear. perhaps addition to storageKind as possible values is meant?

Proposition: close issue and reopen, if the requirement resurfaces

**Comment by AFoletti** [comment]
You are totally right: this is really unclear. I fully agree with your proposition
